package.json

```diff
@@ -54,7 +57,7 @@
   },
   "lint-staged": {
     "*.{ts,tsx,js,json,css}": [
-      "tslint -c tslint.json --fix",
+      "tslint -c tslint.json --fix -e \"grammar.ts\"",
```

Comment:
yes but here husky run linter for each file independently, and exclusion in tslint.config file doesn't much a path

Reply:
since we have it in two places now, maybe just put it inside the file itself?
```
earleyc ./src/query-parser/grammar.ne -o ./src/query-parser/grammar.ts && echo '// tslint:ignore ' > ./src/query-parser/grammar.ts
```
something like this?